light-client/src/errors.rs

```diff
@@ -107,3 +113,15 @@ impl ErrorExt for ErrorKind {
         }
     }
 }
+
+impl<T: Debug + Send + Sync + 'static> From<crossbeam::SendError<T>> for ErrorKind {
+    fn from(_err: crossbeam::SendError<T>) -> Self {
+        Self::ChannelDisconnected
```

Comment:
Why not additionally wrap the underlying error msg? (see above)

Reply:
https://github.com/informalsystems/tendermint-rs/pull/403#discussion_r449255706